src/include/symbol_table/symbol_table.h

```diff
@@ -0,0 +1,138 @@
+#ifndef JUCC_SYMBOL_TABLE_H
+#define JUCC_SYMBOL_TABLE_H
+
```

Comment:
Update header

Reply:
To what ?

Reply:
JUCC_SYMBOL_TABLE_SYMBOL_TABLE_H
Namespaces, filename H